### PR TITLE
Fix: Soft coded all the strings

### LINKED ISSUE #2573: Remove all hardcoded strings
**Describe the chore**
Remove all the hardcoded strings.

**Would you like to work on the issue?**
- [x] Yes
- [ ] No

### PR DESCRIPTION
**Pull request by insaiyancvk**
Fixes #2573 

Changes: 
- Soft coded all the strings in content_event.xml, fragmant_profile.xml, item_social_link.xml, item_speaker.xml and order_completed.xml.

### CODE REVIEW

File: app/src/main/java/org/fossasia/openevent/general/search/SearchResultsViewModel.kt [hunk 1]
```diff
@@ -367,7 +367,6 @@ class SearchResultsViewModel(
             removeFavorite(event)
         }
     }
-
```
Comment (iamareebjamal) [inline app/src/main/java/org/fossasia/openevent/general/search/SearchResultsViewModel.kt]: revert

File: app/src/main/res/layout/item_speaker.xml [hunk 2]
```diff
@@ -30,7 +30,7 @@
             android:id="@+id/speakerNameTv"
             android:layout_width="wrap_content"
             android:layout_height="wrap_content"
-            android:text="Pulkit Aggarwal"
+            android:text="@string/pulkit_aggarwal"
```
Comment (iamareebjamal) [inline app/src/main/res/layout/item_speaker.xml]: No need. revert

File: app/src/main/res/layout/item_speaker.xml [hunk 3]
```diff
@@ -52,7 +52,7 @@
             android:layout_width="wrap_content"
             android:layout_height="wrap_content"
             android:layout_marginTop="@dimen/layout_margin_small"
-            android:text="Coding Blocks"
+            android:text="@string/coding_blocks"
```
Comment (iamareebjamal) [inline app/src/main/res/layout/item_speaker.xml]: no need. revert

File: app/src/main/res/layout/item_recent_search.xml [hunk 4]
```diff
@@ -5,7 +5,7 @@
     android:layout_width="match_parent"
     android:layout_height="wrap_content"
     android:background="?selectableItemBackground"
-    android:padding="@dimen/padding_large">
+    android:padding="16dp">
```
Comment (iamareebjamal) [inline app/src/main/res/layout/item_recent_search.xml:8]: Why?

File: app/src/main/res/values/strings.xml [hunk 5]
```diff
@@ -534,5 +534,12 @@
     <string name="change_password_success_message">Password changed successfully!</string>
     <string name="tickets_for">Ticket(s) for %1$s</string>
     <string name="error_making_paypal_payment_message">Fail on making Paypal payment</string>
+    <string name="tick">tick</string>
+    <string name="general_settings_and_support">General Settings and Support</string>
+<<<<<<< HEAD
+    <string name="sociallink">Social Link</string>
+=======
+    <string name="socialLink">Social Link</string>
+>>>>>>> 51274de059a3850db03a67b047cfc9eb20eaf1e3
```
Comment (iamareebjamal) [inline app/src/main/res/values/strings.xml:546]: nice

Comment (insaiyancvk) [reply app/src/main/res/values/strings.xml:546]: I messed up before committing. I did a git pull before pushing the changes I made and this happened. I would like to close this pull request as I do not know to fix the problem that I am currently facing.

**Pull request closed without merging**